modules/kubernetes-addons/variables.tf

```diff
@@ -965,6 +965,12 @@ variable "external_secrets_helm_config" {
   description = "External Secrets operator Helm Chart config"
 }
 
+variable "external_secrets_irsa_policies" {
```

Comment:
lets provide a default default policy for users, here is one for reference https://github.com/terraform-aws-modules/terraform-aws-iam/blob/master/modules/iam-role-for-service-accounts-eks/policies.tf#L418

Reply:
I added a default policy with parameters to pass arns through.